examples/integrations/databricks/labelbox_databricks_example.ipynb

```diff
@@ -1 +1 @@
milarity search. \n\n<img src=\"https://files.readme.io/14f82d4-catalog-marketing.jpg\" alt=\"Catalog\" width=\"800\"/>\n\n<h3> [Model Diagnostics](https://labelbox.com/product/model-diagnostics) </h3>\nLabelbox complements your MLFlow experiment tracking with the ability to easily visualize experiment predictions at scale. Model Diagnostics helps you quickly identify areas where your model is weak so you can collect the right data and refine the next model iteration. \n\n<img src=\"https://images.ctfassets.net/j20krz61k3rk/4LfIELIjpN6cou4uoFptka/20cbdc38cc075b82f126c2c733fb7082/identify-patterns-in-your-model-behavior.png\" alt=\"Diagnostics\" width=\"800\"/>"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":true,"inputWidgets":{},"nuid":"7f88d026-d4f3-4efc-929a-39f2d5393448"}}},{"cell_type":"markdown","source":["While using the Labelbox Connector for Databricks, you will likely use the Labelbox SDK (e.g. for programmatic ontology creation). These resources will help familiarize you with the Labelbox Python SDK: \n* [Visit our docs](https://labelbox.com/docs/python-api) to learn how the SDK works\n* Checkout our [notebook examples](https://github.com/Labelbox/labelspark/tree/master/notebooks) to follow along with interactive tutorials\n* view our [API reference](https://labelbox.com/docs/python-api/api-reference).\n\n<h4>Questions or comments? Reach out to us at [ecosystem+databricks@labelbox.com](mailto:ecosystem+databricks@labelbox.com)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"More Info","showTitle":true,"inputWidgets":{},"nuid":"8eaf3897-b0a8-44e9-aea4-fe58d5ee246f"}}},{"cell_type":"markdown","source":["Copyright Labelbox, Inc. 2021. The source in this notebook is provided subject to the [Labelbox Terms of Service](https://docs.labelbox.com/page/terms-of-service).  All included or referenced third party libraries are subject to the licenses set forth below.\n\n|Library Name|Library license | Library License URL | Library Source URL |\n|---|---|---|---|\n|Labelbox Python SDK|Apache-2.0 License |https://github.com/Labelbox/labelbox-python/blob/develop/LICENSE|https://github.com/Labelbox/labelbox-python\n|Labelbox Connector for Databricks|Apache-2.0 License |https://github.com/Labelbox/labelspark/blob/master/LICENSE|https://github.com/Labelbox/labelspark\n|Python|Python Software Foundation (PSF) |https://github.com/python/cpython/blob/master/LICENSE|https://github.com/python/cpython|\n|Apache Spark|Apache-2.0 License |https://github.com/apache/spark/blob/master/LICENSE|https://github.com/apache/spark|"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"1e446cb2-0338-4e5a-9d03-fc24901b3109"}}}],"metadata":{"application/vnd.databricks.v1+notebook":{"notebookName":"labelbox_databricks_example","dashboards":[],"notebookMetadata":{"pythonIndentUnit":2},"language":"python","widgets":{},"notebookOrigID":2233145370303228}},"nbformat":4,"nbformat_minor":0}
```

Comment:
I didn't do YAPF on the ipynb file because it's designed to just be rendered/edited in a notebook environment on Databricks. I think this can go through without being YAPF'd

Reply:
Since this is a computer-generated Python notebook file, can we approve it without having YAPF applied? I'd like to avoid changing the files computer-generated by Databricks.

Reply:
Unfortunately if we let this pass without passing YAPF then it will break all future branches/prs. In the end we had to run yapf on the previous pr as well